playbooks/hc-ansible-deployment/gluster_inventory.yml

```diff
@@ -402,3 +405,9 @@ gluster:
   storage_domains:
     - {"name":"data","host":"host1-frontend-network-FQDN","address":"host1-backend-network-FQDN","path":"/data","function":"data","mount_options":"backup-volfile-servers=host2-backend-network-FQDN:host3-backend-network-FQDN"}
     - {"name":"vmstore","host":"host1-frontend-network-FQDN","address":"host1-backend-network-FQDN","path":"/vmstore","function":"data","mount_options":"backup-volfile-servers=host2-backend-network-FQDN:host3-backend-network-FQDN"}
+
+# In case of IPv6 based deployment there is additional mount option required i.e. xlator-option="transport.address-family=inet6"
+# Ex:
+#storage_domains:
```

Comment:
does user have to uncomment this as well to enable the appropriate storage domain to be set?

Reply:
Corrected with next commit